frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -109,18 +109,23 @@ export const MonitoringDashboard: React.FC<Props> = ({ match, timespan, pollInte
         <title>{t('devconsole~Dashboard')}</title>
       </Helmet>
       <div className="odc-monitoring-dashboard">
-        <div className="odc-monitoring-dashboard__dropdown-options">
-          <TimespanDropdown />
-          <PollIntervalDropdown />
-        </div>
-        <div className="row odc-monitoring-dashboard__resource-toolbar">
-          <div className="col-lg-3 col-md-3 col-sm-4 col-xs-6">
+        <div className="odc-monitoring-dashboard__resource-toolbar">
```

Comment:
There is definitely a way to improve this with PF Flex components... ideally we should avoid rolling our own classes and styles.